.kotlin_alpha/services/eventbridge/src/main/kotlin/com/kotlin/eventbridge/PutEvents.kt

```diff
@@ -0,0 +1,80 @@
+// snippet-sourcedescription:[PutEvents.kt demonstrates how to send custom events to Amazon EventBridge.]
+//snippet-keyword:[AWS SDK for Kotlin]
+// snippet-service:[Amazon EventBridge]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[03/04/2021]
+// snippet-sourceauthor:[scmacdon - AWS]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.kotlin.eventbridge
+
+// snippet-start:[eventbridge.kotlin._put_event.import]
+import aws.sdk.kotlin.services.eventbridge.EventBridgeClient
+import aws.sdk.kotlin.services.eventbridge.model.PutEventsRequest
+import aws.sdk.kotlin.services.eventbridge.model.PutEventsRequestEntry
+import aws.sdk.kotlin.services.eventbridge.model.EventBridgeException
+import java.util.ArrayList
+import kotlin.system.exitProcess
+// snippet-end:[eventbridge.kotlin._put_event.import]
+
+suspend fun main(args: Array<String>) {
+
+    val usage =
+        """
+        To run this example, supply two resources, identified by Amazon Resource Name (ARN), which the event primarily concerns. Any number, including zero, may be present. 

```

Comment:
I don't quite understand this comment. Do you mean "supply up to two resources"? If not, I don't understand the "any number, including zero" part.